bumble/smp.py

```diff
@@ -1771,7 +1772,26 @@ def send_command(self, connection: Connection, command: SMP_Command) -> None:
         cid = SMP_BR_CID if connection.transport == BT_BR_EDR_TRANSPORT else SMP_CID
         connection.send_l2cap_pdu(cid, command.to_bytes())
 
+    def on_smp_security_request(
```

Comment:
nit: to match the pattern used elsewhere, we should keep the full name of the command: `on_smp_security_request_command` (like we have `on_smp_pairing_request_command` for example.

Reply:
Done